src/index.html

```diff
@@ -1,13 +1,308 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>THE MET</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&display=swap" rel="stylesheet">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header page__section">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+            <a href="/" class="logo">
+              <img
+                src="./images/THE MET.png"
+                alt="THE MET"
+                class="logo__image"
+              >
+            </a>
+            <div class="nav">
+              <a href="tel:+1 212-923-3700" class="icon icon--phone nav--call"></a>
+              <a href="#menu" class="icon icon--menu"></a>
+            </div>
+          </div>
+          <h1 class="header__title">Welcome <br> to The MET</h1>
+        </div>
+      </div>
+
+    </header>
+
+    <nav class="page__menu menu" id="menu">
```

Comment:
Checklist 11th item, you have horizontal scroll in menu
![image](https://user-images.githubusercontent.com/37161315/170642987-d22895c0-abbc-4f48-a66e-ad410b052a09.png)